designs/vcenter-template-import.md

```diff
@@ -44,28 +53,33 @@ manually imported the OVA into a template in vcenter, or not specify the templat
 field empty if they want it imported via a generated name that is based on the Kubernetes
 version, OS, and hash of the build. 
 
-With this command, if a user has the template name defined, we will check to see if a template of that 
-name already exists and the appropriate tags are set. If the template does not exist, we will import the template under
-that name with the appropriate tags. If the template exists but the tags don't match the bundle 
-that the CLI is configured with, we will error saying that the appropriate tags are not set, allowing
-the user to decide if they want to manually fix the tags or delete the template first before running
-the command again. 
+If the user specifies the cluster spec as input, we will get most of the template configuration
+from there except for the name. If there is a name defined, we will throw an error mentioning
```

Comment:
Should this be only if the full path from the flags doesn't match the one in the spec?
Like the user could just pass the same path to both in spec and flags, and that would work.